addons/binding/org.openhab.binding.meteostick/README.md

```diff
@@ -0,0 +1,22 @@
+#MeteoStick Binding
+
+This is the binding for the [MeteoStick](http://www.smartbedded.com/wiki/index.php/Meteostick) weather receiver dongle.
+
+## Supported Things
+
+This binding support 6 different things types
+
+| Thing | Type    | Description  |
+|----------------|---------|-----------------------------------|
+| bridge | Bridge | This is the MeteoStick USB stick  |
+| meteostick_davis_iss | Thing | This is the Davis Vue ISS |
+
+
+## Binding Configuration
+
+Configure the MeteoStick bridge - the port and frequency band need to be set
+
+Configure the Sensor - the channel number needs to be set.
+
+
```

Comment:
Please enhance the documentation and provide an example configuration  (see [template](https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/README.md))